src/datasets/builder.py

```diff
@@ -210,20 +213,24 @@ def get_exported_dataset_info(self) -> DatasetInfo:
 
     def _create_builder_config(self, name=None, custom_features=None, **config_kwargs):
         """Create and validate BuilderConfig object.
-        Uses the first configuration in self.BUILDER_CONFIGS if name is None
+        Raises ValueError if there are multiple builder configs and name and DEFAULT_CONFIG_NAME are None.
         config_kwargs override the defaults kwargs in config
         """
         builder_config = None
         if name is None and self.BUILDER_CONFIGS and not config_kwargs:
-            if len(self.BUILDER_CONFIGS) > 1:
-                example_of_usage = "load_dataset('{}', '{}')".format(self.name, self.BUILDER_CONFIGS[0].name)
-                raise ValueError(
-                    "Config name is missing."
-                    "\nPlease pick one among the available configs: %s" % list(self.builder_configs.keys())
-                    + "\nExample of usage:\n\t`{}`".format(example_of_usage)
-                )
-            builder_config = self.BUILDER_CONFIGS[0]
-            logger.info("No config specified, defaulting to first: %s/%s", self.name, builder_config.name)
+            if self.DEFAULT_CONFIG_NAME is not None:
+                builder_config = self.builder_configs.get(self.DEFAULT_CONFIG_NAME)
+                logger.info("No config specified, defaulting to: %s/%s", self.name, builder_config.name)
```

Comment:
As we said offline, this should be warning indeed.

The other one on the other hand `No config specified...` should stay at info level since there is no ambiguity in which config to load.